src/internals/channel.js

```diff
@@ -81,26 +82,24 @@ class Channel extends stream.Duplex {
     })
   }
 
-  destroy (err/* : Error */) {
-    this._destroy(err, true)
+  /**
+   * Conditionally emit errors if we have listeners. All other
+   * events are sent to EventEmitter.emit
+   * @param {string} eventName
+   * @param  {...any} args
+   * @returns {void}
+   */
+  emit (eventName, ...args) {
+    if (eventName === 'error' && !this._events.error) {
+      this.log('error', ...args)
+    } else {
+      super.emit(eventName, ...args)
+    }
   }
 
-  _destroy (err/* : Error */, local/* : bool */) {
+  _destroy (err/* : Error */, callback) {
+    const local = this.local
```

Comment:
We only use `local` twice here. Why not use `this.local` in both of them instead of creating this const for replicating it?

Reply:
Good call, I removed `const local` in favor of just using `this.local`.